src/frontend/scss/_main.scss

```diff
@@ -5,7 +5,15 @@
 
 // Our variables must be imported before Bootstrap code so they can override it
 @import './settings/fonts';
-@import './settings/variables';
+@import 'bootstrap/scss/functions';
+@import 'bootstrap/scss/variables';
+
+// Design color stuff
+@import 'colors/palette';
+@import 'colors/gradients';
+@import 'colors/schemes';
+@import 'colors/theme';
+@import 'settings/variables';
```

Comment:
I think you should use relative paths.
https://app.circleci.com/pipelines/github/openfun/richie/4230/workflows/bf557919-0330-4024-9c74-78bd24377ee2/jobs/139916

Reply:
Thanks. I don't why, but the `make build-front` work ok...